pkg/sessions/redis/session_store_test.go

```diff
@@ -41,12 +48,25 @@ var _ = Describe("Redis SessionStore Tests", func() {
 		mr.Close()
 	})
 
+	JustAfterEach(func() {
+		// Release any connections immediately after the test ends
+		if redisStore, ok := ss.(*SessionStore); ok {
+			if redisStore.Client != nil {
+				redisStore.Client.(closer).Close()
```

Comment:
Since `Close()` can error - I think we should assert that it didn't error.

Otherwise someday some developer will run into the same unclear socket timeout/file descriptor errors that are less clear.

Reply:
Good catch, gimme a second